exercises/allergies/test/test_allergies.c

```diff
@@ -10,30 +10,9 @@ void tearDown(void)
 {
 }
 
-//helper functions
-void list_count_is(int count, allergen_list_t * list)
-{
-   TEST_ASSERT_EQUAL(count, list->count);
-}
-
-void list_contains(allergen_t allergen, allergen_list_t * list)
-{
-   bool allergen_found = false;
-
-   for (int i = 0; i < list->count; i++) {
-      if (list->allergens[i] == allergen) {
-         allergen_found = true;
-         break;
-      }
-   }
-
-   TEST_ASSERT_TRUE(allergen_found);
-}
```

Comment:
:+1: 

While I'm normally all for test helpers, I think keeping these tests as simple as possible makes this more accessible.